tests/storage/hotplug.go

```diff
@@ -434,15 +482,15 @@ var _ = SIGDescribe("Hotplug", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		table.DescribeTable("Should add volumes on an offline VM", func(addVolumeFunc func(name, namespace, volumeName, claimName, bus string), removeVolumeFunc func(name, namespace, volumeName string)) {
+		table.DescribeTable("Should add volumes on an offline VM", func(addVolumeFunc addVolumeFunction, removeVolumeFunc removeVolumeFunction) {
 			By("Adding test volumes")
-			addVolumeFunc(vm.Name, vm.Namespace, testNewVolume1, "madeup", "scsi")
-			addVolumeFunc(vm.Name, vm.Namespace, testNewVolume2, "madeup", "scsi")
+			addVolumeFunc(vm.Name, vm.Namespace, testNewVolume1, "madeup", "scsi", false)
+			addVolumeFunc(vm.Name, vm.Namespace, testNewVolume2, "madeup", "scsi", false)
```

Comment:
Both of the dry run arguments are false here, I guess it's a typo :)

Reply:
Yes, the test here is adding and then removing two volumes (`testNewVolume1` and `testNewVolume2`), without dryrun :)